### PR TITLE
Add kubernetes deployment for celery workers

### LINKED ISSUE #103: Kubernetes deployment for Celery workers
Create a Kubernetes deployment for the `conductor` (https://github.com/golemfactory/concent/issues/207) and `concent-api` apps started with `celery worker`. This is a part of the implementation of the https://github.com/golemfactory/concent/issues/193 blueprint.

- Define Kubernetes deployments
    - Define resource limits for the image.
    - Run one worker instance in each container
    - Each deployment should have three pods.
    - The pod should be running `celery worker` command with appropriate parameters inside a container started from the `concent-api` image.
- Add separate `local_settings.py` for each app, each based on the one that's used to run a Django server:
    - Each one should connect to the database on its own cluster
    - `ALLOWED_HOSTS` can be empty (it won't be running a web server).
    - `CONCENT_FEATURES`: set to `['conductor-worker']` for Conductor worker and `['concent-worker']` for Concent worker.
    - Broker address: the address of RabbitMQ on the storage cluster (#88). Internal address for both for now but when we set up separate clusters, Concent worker will need the external address.

### PR DESCRIPTION
**Pull request by bartoszbetka**
Resolves #103
It's depend on issue #109, [#215](https://github.com/golemfactory/concent/issues/215), [#224](https://github.com/golemfactory/concent/issues/224)

### CODE REVIEW

File: kubernetes/config-maps/conductor-worker/local_settings.py.j2 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    'environment': '{{ gke.cluster }}',
+    'tags': {'component': 'conductor-worker'},
+}
+
+CELERY_BROKER_URL = 'amqp://rabbitmq.default.svc.cluster.local:5672'
+
+EMAIL_SUBJECT_PREFIX = "[{{ gke.cluster }}:conductor-worker] "
+
+CONCENT_FEATURES = ["conductor-worker"]
+
```
Comment (cameel) [inline kubernetes/config-maps/conductor-worker/local_settings.py.j2]: Please remove this empty line.

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from ..production import *
+
+from .email_settings import *
+from .timing_settings import *
+
+# Create secrets.py and put passwords and private keys in it.
+# Don't commit it into the repository.
+from ..secrets.secrets import (
+    SECRET_KEY,
+    DATABASE_PASSWORD,
```
Comment (cameel) [inline kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: Let's rename this secret to `CONTROL_DATABASE_PASSWORD`.

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from ..production import *
+
```
Comment (cameel) [inline kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: There's no `ALLOWED_HOSTS` defined in this file. Are you sure it's going to work? I think that this setting must be defined for the `manage.py check` to pass, even if it's empty.

File: kubernetes/services/concent-api-worker.yml.j2 [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          imagePullPolicy:  Always
+          command:          ["/usr/local/bin/run-celery.sh"]
+          ports:
+            - containerPort: 80
+          resources:
+            requests:
+              memory: "0.6Gi"
+              cpu:    "50m"
+            limits:
+              memory: "1Gi"
```
Comment (cameel) [inline kubernetes/services/concent-api-worker.yml.j2]: These memory limits are pretty high. Can we set them lower?

File: containers/concent-api/run-celery.sh [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+cd "/srv/http/concent_api"
+
+exec "/srv/http/virtualenv/bin/celery" worker    \
+    --app=concent_api                            \
+    --loglevel=INFO
```
Comment (cameel) [inline containers/concent-api/run-celery.sh]: Please rebase this on master. I think we added parameters to this script in another pull request.

File: kubernetes/config-maps/concent-api-worker/local_settings.py.j2 (hunk 3 above)
Comment (bartoszbetka) [reply kubernetes/config-maps/concent-api-worker/local_settings.py.j2]: We don't use it at all in `concent`. The `python manage.py check` command pass without this setting on local environment.

File: kubernetes/services/concent-api-worker.yml.j2 (hunk 4 above)
Comment (bartoszbetka) [reply kubernetes/services/concent-api-worker.yml.j2]: Ok. I change them to the same resources as `conductor`.

File: containers/concent-api/run-celery.sh (hunk 5 above)
Comment (bartoszbetka) [reply containers/concent-api/run-celery.sh]: Done.

**Pull request merged**